homeassistant/components/blastbot_cloud/climate.py

```diff
@@ -0,0 +1,186 @@
+"""Platform for AC integration."""
+from typing import List
+
+from blastbot_cloud_api.models.control import Control
+
+from homeassistant.components.climate import ClimateEntity
+from homeassistant.components.climate.const import (
+    FAN_AUTO,
+    FAN_HIGH,
+    FAN_LOW,
+    FAN_MEDIUM,
+    HVAC_MODE_COOL,
+    HVAC_MODE_OFF,
+    SUPPORT_FAN_MODE,
+    SUPPORT_TARGET_TEMPERATURE,
+)
+from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS
+
+from .const import DOMAIN
+
+
+async def async_setup_entry(hass, entry, async_add_entities):
+    """Set up the Blastbot Cloud platform."""
+
+    # Get API object from data
+    api = hass.data[DOMAIN][entry.entry_id]
+
+    # Add switches
+    acs = await api.async_get_acs()
+    async_add_entities(BlastbotACEntity(ac) for ac in acs)
+
+
+class BlastbotACEntity(ClimateEntity):
+    """Representation of a Blastbot AC."""
+
+    def __init__(self, control: Control) -> None:
+        """Initialize the AC."""
+        self._control = control
+
+    @property
+    def unique_id(self) -> str:
+        """Return a unique, Home Assistant friendly identifier for this entity."""
+        return f"{DOMAIN}_c{self._control.id}"
```

Comment:
Don't include the domain in the unique id. The entity registry is aware of both integration domain and platform domain. Just use the control id. If it's a string we don't need an f-string either.
```suggestion
        return self._control.id
```